parquet-plugins/parquet-encoding-vector/src/test/java/org/apache/parquet/column/values/bitpacking/TestByteBitPacking512VectorLE.java

```diff
@@ -0,0 +1,172 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.parquet.column.values.bitpacking;
+
+import org.junit.Test;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.math.BigDecimal;
+import java.nio.ByteBuffer;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertArrayEquals;
+
+public class TestByteBitPacking512VectorLE {
+  private static final Logger LOG = LoggerFactory.getLogger(TestByteBitPacking512VectorLE.class);
+  @Test
+  public void unpackValuesUsingVector() {
+    if (ParquetReadRouter.getSupportVectorFromCPUFlags() != VectorSupport.VECTOR_512) {
+      LOG.info("avx512vbmi and avx512_vbmi2 are not supported, skip this test.");
+      return;
+    }
```

Comment:
This way this is not a quality gate. I think we should not allow bypassing this test here but somehow ensure that the environment is capable of executing it. What do you think?

Reply:
@gszadovszky I agree with you. 
1、I have verified with `lscpu command` that there are not avx512vbmi and avx512_vbmi2 instruction set on actions runner.
2、I have checked the docs(https://docs.github.com/en/actions/using-github-hosted-runners/about-github-hosted-runners), github actions do not support to select runner with specific instruction set.
3、I have resubmitted the help on how to select specific runner(https://github.com/orgs/community/discussions/48955)
I think there are two ways to fix it:
a. skip vector related tests until github actions support to select specific runner.
b. I will add a self-hosted runner for parquet-mr repo with cloud instance  to run vector related tests. But the runner is not long time since I have to pay money for it.

What do you think ?  @gszadovszky

Reply:
@jiangjiguang, I wouldn't expect you to spend money on it. I accept there is no proper solution for now. Since this is more for the developers who work on this part and it does not part of the official parquet-mr distribution I am fine with not enforcing these tests to be executed.
I would suggest to use `Assume` from `junit`. This way the tests would be marked as _skipped_ instead of having them _passed_.

Reply:
@gszadovszky thanks, I have used assume from junit and added vector-plugins in vector-plugins.yml.

please take a look.